src/Extend/ApiSerializer.php

```diff
@@ -145,10 +167,10 @@ public function extend(Container $container, Extension $extension = null)
             };
         }
 
-        foreach ($this->mutators as $mutator) {
-            $mutator = ContainerUtil::wrapCallback($mutator, $container);
+        foreach ($this->attributes as $callback) {
+            $callback = ContainerUtil::wrapCallback($callback, $container);
 
-            AbstractSerializer::addMutator($this->serializerClass, $mutator);
+            AbstractSerializer::addMutator($this->serializerClass, $callback);
```

Comment:
do we want to rename the method as well as the property in `AbstractSerializer` while we're at it as well ?

Reply:
Yep! Especially since that's not really public API.